kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriberServiceImpl.java

```diff
@@ -33,7 +40,32 @@ public void add(Subscriber subscriber) {
 
     @Override
     public void update(Subscriber subscriber) {
```

Comment:
There is a test for Transactional in mds.  Which annotation are you using?  org.springframework.transaction.annotation.Transactional?  You will also need to add imports to your pom in the felix bundler section